simplified-books-covers/src/main/java/org/nypl/simplified/books/covers/BookCoverGenerator.java

```diff
@@ -81,19 +77,20 @@ public BookCoverGenerator(
       final String author_maybe = params.get("author");
       final String author = author_maybe == null ? "" : NullCheck.notNull(author_maybe);
 
+      if (width == 0) {
+        width = (int) Math.round(height * .75);
+      }
+      if (height == 0) {
+        height = (int) Math.round(width / .75);
+      }
+
       final TenPrintInputBuilderType ib = TenPrintInput.newBuilder();
       ib.setAuthor(author);
       ib.setTitle(title);
       ib.setCoverHeight(height);
       final TenPrintInput i = ib.build();
       final Bitmap cover = this.generator.generate(i);
-      final Bitmap container = Bitmap.createBitmap(width, height, Config.RGB_565);
```

Comment:
This block of code was adding white margins to the sides of the generated cover images. This looked weird inside the CardView so I removed it. Happy to consider other suggestions.

### Screens

- [Before](https://user-images.githubusercontent.com/107117/73720160-2c653f80-46d6-11ea-8940-5225cc3f5144.png)
- [After](https://user-images.githubusercontent.com/107117/73720162-2c653f80-46d6-11ea-9ff7-0940c5957240.png)

Reply:
I forget what the purpose of this was. Looks like it was just there to ensure the image had the expected size, and was in `RGB_565`.

Reply:
Actually, I've got a feeling that this was a leftover from having to fight with a `HorizontalScrollView` inside a `ListView`. Thankfully we no longer have that problem!

Reply:
Makes sense. I figured it was just adding some padding to the sides of the image.